aetherpy/tests/test_plot_prep.py

```diff
@@ -0,0 +1,226 @@
+#!/usr/bin/env python
+# Copyright 2021, the Aether Development Team (see doc/dev_team.md for members)
+# Full license can be found in License.md
+"""Unit tests for plot data preparation utilities."""
+
+import logging
+import numpy as np
+import pytest
+
+from aetherpy.plot import data_prep
+
+
+class TestDataPrep(object):
+    """Unit tests for data preparation functions."""
+
+    def setup(self):
+        """Initialize clean test environment."""
+
+        self.in_coords = {"lon": np.arange(-180, 180, 1.0),
+                          "lat": np.arange(-89, 90, 0.5),
```

Comment:
Why shouldn't this go from -90 to 90?

Reply:
This goes from -89 to 89.5 because the model doesn't go up to +/-90, it stops beforehand.  I believe that longitude can't be defined at +/- 90 degrees and that's why it isn't included.